app/views/categories/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>New category</h1>
+
+<%= render "form_add", user: @user, category: @category %>
+
+<br>
```

Comment:
[REQUIRED] Please include the back button as stated by the previous reviewer

Reply:
Done @fobadara